rootfs/templates/wrapper

```diff
@@ -88,25 +88,40 @@ function use() {
   if [ -n "${LOCAL_HOME}" ]; then
     local_home=${LOCAL_HOME}
   elif [[ $(uname -r) =~ Microsoft$ ]]; then
-    windows_user_name=$(/mnt/c/Windows/System32/cmd.exe /c 'echo %USERNAME%' | tr -d '\r')
+    
+    wsl_config=`cat /etc/wsl.conf`
```

Comment:
use `$()` instead ` `` `
[Why is $(...) preferred over backticks?](http://mywiki.wooledge.org/BashFAQ/082)